content/influxdb/cloud-dedicated/reference/internals/security.md

```diff
@@ -222,12 +237,78 @@ Users can configure the following security controls:
+- The InfluxDB Cloud Dedicated **cluster URL**
+- A password reset email for setting the login password
+
+With a valid password, the user can login via InfluxData's `influxctl` command line tool.
+The login command initiates an Auth0 browser login so that the password is never exchanged with `influxctl`.
+With a successful authentication to Auth0, InfluxDB Cloud Dedicated provides the user's `influxctl` session with a short-lived [management token](#management-tokens) for access to the Granite service.
+The user interacts with the `influxctl` command line tool to manage the workload cluster, including creating [database tokens](#database-tokens) for database read and write access.
+
+#### Management tokens
+
+Management tokens authenticate user accounts to the Granite service and provide authorizations for workload cluster management activities, including:
+ 
+ - account, pricing, and infrastructure management
+ - inviting, listing, and deleting users
+ - creating, listing, and deleting, and databases
+ - creating, listing, and deleting database tokens
+
+Management tokens consist of the following:
+
+- An access token string (sensitive)
+- A permissions set for management activities (configured by InfluxData during initial provisioning)
+- A mandatory 1 hour expiration
+
+When a user issues a command using the `influxctl` command-line tool, `influxctl` sends the management token string with the request to the server, where Granite validates the token (for example, using Auth0).
+If the management token is valid and not expired, the service then compares the token's permissions against the permissions needed to complete the user's request.
+
+Only valid unexpired tokens that have the necessary permissions sets are authorized to perform management functions with InfluxDB Cloud Dedicated.
+Following security best practice, management tokens are never stored on InfluxDB Cloud Dedicated (Granite or workload cluster) servers, which prevents token theft from the server.
+On the client (the user's system), the management token is stored on disk with restricted permissions for `influxctl` to use on subsequent runs.
+For example, a user's Linux system would store the management token at  `~/.cache/influxctl/*.json` with `0600` permissions (that is, owner read and write, and no access for _group_ or _other_).
+
+#### Database tokens
+
+Database tokens provide authorization for users and client applications to read and write data and metadata in an InfluxDB Cloud Dedicated database.
+All data write and query API requests require a valid database token with sufficient permissions.
+_**Note:** an all-access management token can't read or write to a database because it's not a database token._
+
+Database tokens consist of the following:
+
+- An identifier
+- A description
+- A permissions set for reading from a database, writing to a database, or both
+- A base64-encoded 512-bit random string (the API key string)
```

Comment:
The API string is actually `apiv1_<base64>` so it might be more accurate to say:

- An API key string (format is `apiv<N>_<base64-encoded 512-bit random string>`)